src/os_posix/os_fs.c

```diff
@@ -681,12 +681,19 @@ __posix_open_file(WT_FILE_SYSTEM *file_system, WT_SESSION *wt_session,
 	 * interesting.
 	 */
 	if (!pfh->direct_io && file_type == WT_FS_OPEN_FILE_TYPE_DATA) {
```

Comment:
I would quite like to skip the call to posix_fadvise completely in the default, or normal case. Then we wouldn't override external configurations (I don't know if there is such a facility?).